pkgs/development/python-modules/django-parler/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, django
+, fetchFromGitHub
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "django-parler";
+  version = "2.3";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "django-parler";
+    repo = "django-parler";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-tRGifFPCXF3aa3PQWKw3tl1H1TY+lgcChUP1VdwG1cE=";
+  };
+
+  propagatedBuildInputs = [
+    django
+  ];
+
+  #tests depricated
+  doCheck = false;
```

Comment:
please try to run the tests https://github.com/django-parler/django-parler/tree/master/parler/tests